pkgs/by-name/ti/tinysparql/package.nix

```diff
@@ -87,9 +86,6 @@ stdenv.mkDerivation (finalAttrs: {
       avahi
       libstemmer
       dbus
```

Comment:
By the way, dbus is probably optional as well: https://github.com/GNOME/tinysparql/blob/8b3ceebb90dcd0d874485fbe46234e26213b6309/meson.build#L277